src/FondOfSpryker/Zed/CategoryExtendStorage/Business/Storage/CategoryNodeExtendStorage.php

```diff
@@ -132,10 +133,17 @@ protected function mapToCategoryNodeStorageTransfer(
             $includeParents
         );
 
-        foreach ($this->storageMapperExpanderPlugins as $storageExpanderPlugin) {
-            $storageExpanderPlugin->expand($categoryNodeStorageTransfer, $categoryNode);
-        }
+        try {
+            /** @var \Orm\Zed\Category\Persistence\SpyCategoryAttribute $attribute */
+            $attribute = $categoryNode->getCategory()->getAttributes()->getFirst();
+
+            foreach ($this->storageMapperExpanderPlugins as $storageExpanderPlugin) {
+                $storageExpanderPlugin->expand($categoryNodeStorageTransfer, $categoryNode, $attribute);
+            }
 
-        return $categoryNodeStorageTransfer;
+            return $categoryNodeStorageTransfer;
+        } catch (PropelException $e) {
+            return $categoryNodeStorageTransfer;
+        }
     }
```

Comment:
Hast du überlegt vielleicht den error zu unterdrücken und dann auf null zu prüfen mit early return?

Reply:
Er will ja in erster Linie getCategory Exception abfangen